modules/kraken2.nf

```diff
@@ -57,6 +57,7 @@ process KRAKEN2_BUILD_LIBRARY {
 process KRAKEN2_BUILD {
 
     container 'quay.io/biocontainers/kraken2:2.1.2--pl5321h9f5acd7_2'
+    errorStrategy = { task.attempt <= 3 ? 'retry' : 'finish' }
```

Comment:
Why don't we move this to the modules.config?

I think we can abstract this with labels.. there are some 2 attempts and other with 3.